src/components/RecoilAppController/RecoilAppController.tsx

```diff
@@ -164,6 +167,10 @@ export const RecoilAppController = () => {
         visible={circleSelectorOpen}
       />
       <LoadingModal text={globalLoadingText} visible={globalLoading > 0} />
+      <EditProfileModal
```

Comment:
Hm, I want to defer that, I will likely move this sort of thing around, because I am leaning towards writing some sort of event bus. and I can separate these global UI elements from 


`{editProfileOpen && <EditProfileModal...>}`
hm, but that is what <Modal> is for. Probably there is a mounting unmounting cost. I just ran the profiler and this component was in suspense the entire time I changed between pages..

I think it is performance paranoia

Reply:
> I think it is performance paranoia

yeah, you might be right ¯\_(ツ)_/¯